server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/RemoveUserAction.java

```diff
@@ -83,6 +86,17 @@ public void handle(Request request, Response response) throws Exception {
     }
   }
 
+  /**
+   * Verify that there are still users with admin global permission if user is removed from the group.
+   */
+  private void checkNotTryingToDeleteLastAdmin(DbSession dbSession, GroupId group, UserDto user) {
```

Comment:
since the method raises an exception (rather than returning a boolean which is usually implied by the method name check*), you could name it `ensureLastAdminIsNotRemoved` or something like that